### PR TITLE
Threadpool executor

### PR DESCRIPTION
**Pull request by justinGilmer**
Using concurrent.futures.ThreadPoolExecutor to speed up streamset calls.

Includes recompiled protobuf files as well.

### CODE REVIEW

File: btrdb/transformers.py [hunk 1]
```diff
@@ -188,8 +188,9 @@ def to_array(streamset, agg="mean"):
                 segment.append(point.value)
             else:
                 segment.append(getattr(point, agg))
-        results.append(segment)
-    return np.array(results)
+        results.append(np.array(segment))
+    print(results)
```
Comment (andrewchambers) [inline btrdb/transformers.py:192]: Looks like an accident

File: .idea/btrdb-python.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (andrewchambers) [inline .idea/btrdb-python.iml:1]: Not sure we want to commit the .idea files, not sure what the best practice is.

Comment (justinGilmer) [reply .idea/btrdb-python.iml:1]: agreed. Got past me in the file review.

File: btrdb/transformers.py (hunk 1 above)
Comment (justinGilmer) [reply btrdb/transformers.py:192]: Indeed. cleaning now.

**Pull request closed without merging**